com.ekitabu.eKitabu.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "com.ekitabu.eKitabu",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "20.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "ekitabu",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-documents:ro",
+        "--filesystem=xdg-videos:ro",
+        "--filesystem=xdg-download",
+        "--talk-name=com.canonical.AppMenu.Registrar"
+    ],
+    "modules": [
+        {
+            "name": "ekitabu",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install eKitabu_4.5.7_amd64.deb ${FLATPAK_DEST}/bin/ekitabu",
+                "install -Dm644 com.eKitabu.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml",
+                "install -Dm644 eKitabu.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.eKitabu.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "eKitabu.png"
+                },
+                {
+                    "type": "file",
+                    "filename": "./eKitabu_4.5.7_amd64.deb",
+                    "url": "https://github.com/ekitabu-LLC/flathub/releases/download/v4.0.1/eKitabu_4.5.7_amd64.deb",
+                    "sha256": "1460e1b02d300b3cb98299d407ed9bed5b96357f375e71dacb1ba7300e989202",
+                    "size": 76245298,
```

Comment:
Size is only needed for extra-data sources. Deleting this line will address the following warning from `flatpak-builder`:

```
(flatpak-builder:7119): flatpak-builder-WARNING **: 11:02:54.783: Unknown property size for type BuilderSourceFile
```

```suggestion
```